mrp_production_auto_validate/models/mrp_production.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from odoo import _, api, exceptions, fields, models, tools
+
+_logger = logging.getLogger(__name__)
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    auto_validate = fields.Boolean(
```

Comment:
Shouldn't this field be related to the field on the BOM or at least computed? Relying on the onchange mechanism means it will be set to True only if the BOM is set on the Form view.